ale/drivers/kaguya_driver.py

```diff
@@ -0,0 +1,96 @@
+import spiceypy as spice
+import os
+import pvl
+import ale
+
+import numpy as np
+from ale.drivers.base import Driver, LineScanner, PDS3, Spice, TransverseDistortion
+
+
+class TcPds3Driver(Driver, LineScanner, PDS3, Spice, TransverseDistortion):
+    @property
+    def instrument_id(self):
+        instrument = self.label.get("INSTRUMENT_ID")
+        if instrument == "TC1":
+            return "LISM_TC1"
+        elif instrument == "TC2":
+            return "LISM_TC2"
+
+
+    @property
+    def metakernel(self):
+        self._metakernels = 'mk_kaguyatc.tm'
+        return self._metakernels
+
+
+    @property
+    def starting_ephemeris_time(self):
+        if not hasattr(self, '_starting_ephemeris_time'):
+            self._starting_ephemeris_time = self.label.get('CORRECTED_SC_CLOCK_START_COUNT').value
+            self._starting_ephemeris_time = spice.sct2e(self.spacecraft_id, self._starting_ephemeris_time)
+        return self._starting_ephemeris_time
+
+
+    @property
+    def _detector_center_line(self):
+        return 1
+
+    @property
+    def _detector_center_sample(self):
+        return spice.gdpool('INS{}_CENTER'.format(self.ikid), 0, 2)[0]
+
+
+    @property
+    def _sensor_orientation(self):
+        if not hasattr(self, '_orientation'):
+            current_et = self.starting_ephemeris_time
+            qua = np.empty((self.number_of_quaternions, 4))
+            for i in range(self.number_of_quaternions):
+                # Find the rotation matrix
+                camera2bodyfixed = spice.pxform("J2000",
```

Comment:
Is the sensor position being stored in J2000 or should this be from the sensor (or spacecraft) frame?